android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -381,6 +383,14 @@ public void initializeState() {
         BraveSearchEngineUtils.initializeBraveSearchEngineStates(getTabModelSelector());
     }
 
+    public int getLastTabId() {
+        return mLastTabId;
+    }
+
+    public void setLastTabId(int lastTabId) {
```

Comment:
Does this get called on browser startup to set the currently active tab?

Reply:
@tmancey yes, once news in on.